collectors/python.d.plugin/edgex/edgex.chart.py

```diff
@@ -0,0 +1,312 @@
+# -*- coding: utf-8 -*-
+# Description: EdgeX Platform netdata python.d module
+# Author: Odysseas Lamtzidis (OdysLam)
+# SPDX-License-Identifier: GPL-3.0-or-later
+# Module Version: 0.2
+
+import json
+import threading
+
+from collections import namedtuple
+
+try:
+    from Queue import Queue
+except ImportError:
+    from queue import Queue
+
+from bases.FrameworkServices.UrlService import UrlService
+
+METHODS = namedtuple('METHODS', ['get_data', 'url', 'run'])
+
+ORDER = [
+    'events_throughput',
+    'readings_throughput',
+    # 'events_count',
```

Comment:
i suggest to remove commented out code if it is not needed

Reply:
Agreed